roles/conversion_host/tasks/main.yml

```
@@ -51,7 +51,7 @@
 - name: Download VMware VIX DiskLib
   ansible.builtin.get_url:
     url: "{{ conversion_host_vmware_vix_disklib_url }}/{{ conversion_host_vmware_vix_disklib }}"
-    dest: "/tmp/{{ vmware_vix_disklib }}"
+    dest: "/tmp/{{ vmware_vix_disklib | default(conversion_host_vmware_vix_disklib) }}"
```

```suggestion
    dest: "/tmp/{{ conversion_host_vmware_vix_disklib }}"
```

I would fix it in this way since `vmware_vix_disklib` variable doesn't exists.